### PR TITLE
Fixes #18849: katello-certs-check does not check expiration date

### PR DESCRIPTION
**Pull request by Aftermath**
Added checks for the expiration date of the CA and the Certificate individually.
Had them checked separately as they can be created/developed separately in user environments.

### CODE REVIEW

File: bin/katello-certs-check [hunk 1]
```diff
@@ -53,6 +53,27 @@ function success () {
     echo "[OK]"
 }
 
+function check-expiration () {
+  DATE=$(date -u +"%b %-d %R:%S %Y")
```
Comment (ekohl) [inline bin/katello-certs-check]: Please match the indenting of other functions

Comment (Aftermath) [inline bin/katello-certs-check]: Sorry, missed that.
I will submit that in the next commit.
Looking over some other methods of verification before I finalize.

**Comment by Aftermath** [comment]
Tried to look into the feasibility of getting the expiration from concatenated CA's that may include intermediates, but this would not be simply accomplished by `openssl` commands, we would need to separate the certs out and read them individually by some bash trickery, and I don't this its completely necessary. It seems more likely that the base certificate or CA that actually signed the cert will expire prior to the intermediate or Root CA above them, making this a unlikely case.

**Comment by ehelms** [comment]
Thanks @Aftermath !

**Pull request merged**